examples/units/basic_units.py

```diff
@@ -380,4 +380,17 @@ def cos(x):
         return math.cos(x.convert_to(radians).get_value())
 
 
-units.registry[BasicUnit] = units.registry[TaggedValue] = BasicUnitConverter()
+def register_units():
```

Comment:
I'm still a little tan about who is incorrect here.  It really seems sphinx gallery should spawn a new process for each example so the import happens again.  However, I can see why that would be slow.  It does make me extremely suspicious about changing behaviour via a registry that gets set at import time like this... (and yes we do it internally for dates and categorical as well).